birdwatcher.py

```diff
@@ -47,6 +52,11 @@ def parse_show_protocols(f):
         yield ('bird_up{bird_protocol_instance="%s"}' % protocol_instance,
                int(line[3] == 'up'))
 
+        info = line[5] if len(line)>5 else "unknown"
+        yield bird_info_line(protocol_instance, info, "unknown")
```

Comment:
Looking at the Bird source code, there are more states than just Active and Established:

```
static char *bgp_state_names[] = { "Idle", "Connect", "Active", "OpenSent", "OpenConfirm", "Established", "Close" };
```

Maybe it makes more sense not to add any state names to the source code of this exporter, but simply to export just a single metric containing the literal value of the state?